nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/StandardNiFiServiceFacade.java

```diff
@@ -1462,6 +1459,25 @@ private Set<AffectedComponentEntity> getComponentsAffectedByParameterContextUpda
         return dtoFactory.createAffectedComponentEntities(affectedComponents, revisionManager);
     }
 
+    private void getAffectedComponentsByParameterUpdate(final ParameterContextDTO parameterContextDto, final Set<String> updatedParameterNames, final Set<ComponentNode> affectedComponents,
```

Comment:
We should not have a getter that doesn't return a value. Is the intent to populate the provided `affectedComponents` set? If so, perhaps that should be returned instead.

Reply:
This method has been removed.